libexec/scoop-uninstall.ps1

```diff
@@ -51,72 +51,76 @@ if (!$apps) { exit 0 }
+            continue
+        }
 
-    $manifest = installed_manifest $app $version $global
-    $install = install_info $app $version $global
-    $architecture = $install.architecture
+        $manifest = installed_manifest $app $version $global
+        $install = install_info $app $version $global
+        $architecture = $install.architecture
 
-    run_uninstaller $manifest $architecture $dir
-    rm_shims $manifest $global $architecture
-    rm_startmenu_shortcuts $manifest $global $architecture
+        run_uninstaller $manifest $architecture $dir
+        rm_shims $manifest $global $architecture
+        rm_startmenu_shortcuts $manifest $global $architecture
 
-    # If a junction was used during install, that will have been used
-    # as the reference directory. Otherwise it will just be the version
-    # directory.
-    $refdir = unlink_current $dir
+        # If a junction was used during install, that will have been used
+        # as the reference directory. Otherwise it will just be the version
+        # directory.
+        $refdir = unlink_current $dir
 
-    uninstall_psmodule $manifest $refdir $global
+        uninstall_psmodule $manifest $refdir $global
 
-    env_rm_path $manifest $refdir $global $architecture
-    env_rm $manifest $global $architecture
+        env_rm_path $manifest $refdir $global $architecture
+        env_rm $manifest $global $architecture
 
-    try {
-        # unlink all potential old link before doing recursive Remove-Item
-        unlink_persist_data $dir
-        Remove-Item $dir -Recurse -Force -ErrorAction Stop
-    } catch {
-        if (Test-Path $dir) {
-            error "Couldn't remove '$(friendly_path $dir)'; it may be in use."
-            continue
+        try {
+            # unlink all potential old link before doing recursive Remove-Item
+            unlink_persist_data $dir
+            Remove-Item $dir -Recurse -Force -ErrorAction Stop
+        } catch {
+            if (Test-Path $dir) {
+                Get-error "Couldn't remove '$(friendly_path $dir)'; it may be in use."
+                continue
+            }
         }
     }
-
     # remove older versions
-    $old = Get-InstalledVersion -AppName $app -Global:$global
-    foreach ($oldver in $old) {
-        Write-Host "Removing older version ($oldver)."
-        $dir = versiondir $app $oldver $global
+    $oldVersions = @(Get-ChildItem $appDir -Name -Exclude 'current')
+    foreach ($version in $oldVersions) {
+        Write-Host "Removing older version ($version)."
+        $dir = versiondir $app $version $global
         try {
             # unlink all potential old link before doing recursive Remove-Item
             unlink_persist_data $dir
             Remove-Item $dir -Recurse -Force -ErrorAction Stop
         } catch {
-            error "Couldn't remove '$(friendly_path $dir)'; it may be in use."
+            Get-error "Couldn't remove '$(friendly_path $dir)'; it may be in use."
```

Comment:
Reason of renaming to `Get-error`?

Reply:
Wait, isn't `Get-Error` a PowerShell builtin? I think this was added by mistake, probably due to VSCode's PowerShell extension.

Reply:
Yes, this is a mistake, I'll fix them.

This type of error should be fixed by renaming all `error` in the code base.